src/utils/clip_utils.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
-from numba import vectorize, bool_, float64
+from numba import jit
+import numba
 from shapely.geometry import Polygon
 import pyclipper
 
```

Comment:
Waar heb je deze code vandaan? Opzich kan de tekst van "https://github.com/brycefrank/pyfor/blob/master/pyfor/clip.py" wel weg als we nu deze gebruiken

Reply:
We gebruiken van hun nog steeds square_clip en poly_clip